src/QtLocationPlugin/QGeoMapReplyQGC.cpp

```diff
@@ -135,11 +142,19 @@ QGeoTiledMapReplyQGC::networkReplyFinished()
         }
         emit terrainDone(a, QNetworkReply::NoError);
     } else {
-        //-- This is a map tile. Process and cache it if valid.
-        setMapImageData(a);
-        if(!format.isEmpty()) {
-            setMapImageFormat(format);
-            getQGCMapEngine()->cacheTile(getQGCMapEngine()->urlFactory()->getTypeFromId(tileSpec().mapId()), tileSpec().x(), tileSpec().y(), tileSpec().zoom(), a, format);
+        if (a == _bingNoTileImage) {
```

Comment:
This is an expensive comparison. Maps used to have an ID. Not sure how the new code handles it. The id is in `tileSpec().mapId()`. If simple, I would make sure it's Bing before doing a byte array comparison.

Reply:
Cap, I meant to do that then forgot.